yurt/yurt_core/setup.py

```diff
@@ -55,15 +55,15 @@ def enable_git_repo(config_settings, project_name):
     os.chdir(current_path)
 
 
-def add_all_files_to_git_repo(config_settings, project_name):
+def add_all_files_to_git_repo(_, project_name):
```

Comment:
Probably not the scope of this PR, but if we're not using `config_settings` why is it being passed in here?

Likely we're calling a series of functions in a row with these parameters? Some common solutions would be to use kwargs** or turn this into a class that just has `config_settings` and `project_name` stored on `self` that can be accessed from each method if it needs it.